packages/cloud-config/README-cloud-config-boot-assessment.md

```diff
@@ -10,22 +10,22 @@ The boot assessment process works in the following way:
 
 The grub sentinel files are in the COS_STATE partition, and get installed automatically after reset, install and upgrade:
 
-- `/boot_assessment` - contains the GRUB env sentinel variables
-- `/grub_boot_assessment` - contains the GRUB logic for booting into fallback
+- `/grub2/boot_assessment` - contains the GRUB env sentinel variables
+- `/grub2/grub_boot_assessment` - contains the GRUB logic for booting into fallback
 
-Note: The extra GRUB logic is installed and sourced into `/grubcustom` inside `COS_STATE`.
+Note: The extra GRUB logic is installed and sourced into `/grub2/grubcustom` inside `COS_STATE`.
 
 When a boot failure is detected and the fallback is automatically selected, it will be created a `/run/cos/upgrade_failure` sentinel file during the boot process, which is accessible under the `boot` cloud-init stage.
 
-To enable boot assessment besides upgrades, `enable_boot_assessment_always` can be set to `yes` in the grub environment ( for example in `/oem/grubenv` ).
+To enable boot assessment besides upgrades, `enable_boot_assessment_always` can be set to `yes` in the grub environment ( for example in `/run/cos-state/grub2/grubenv` ).
```

Comment:
I think `/run/cos-state/grub2` is wrong, should be `/run/initramfs/cos-state`, however this is a runtime path, we should not relay on this on our docs. Also there is a long standing TODO in elemental-cli to change this default to be aligned with other paths under `/run/cos/....`

The same I would move the enablement/disable docs to an upgrade/install hook.